### ISSUE TITLE
Exception when animating width & height setters

### ISSUE THREAD

**Issue by jcorentin** [svgdotjs/svg.js#675]
Hi,

Using version 2.6.1.
Trying to animate `width()` & `height()` setters results in an exception : 

> shape.animate().width is not a function

Animating `size()` and `attr('height', ...)` works just fine.

```
var shape = mySVG.rect(200, 200);
shape.animate().width(400); // Exception
```

**Comment by jcorentin** [comment]
`width()` and `height()` are not described as SVG.FX methods in `src/fx.js`. I'm opening a pull request to solve this.

**Comment by dotnetCarpenter** [comment]
Fixed in [2.6.2](/svgdotjs/svg.js/releases/tag/2.6.2)
